### PR TITLE
Update Docs

### PR DESCRIPTION
**Pull request by blisc**
Updated requirements for docs
Updated mocks
Add an `update_docs_docker.sh` that creates docs inside a python3.7 environment

Creating docs in python3.6 results in errors, docs need to be generated with python3.7

@tkornuta-nvidia @DEKHTIARJonathan

### CODE REVIEW

File: docs/docs_zh/sources/source/index.rst [hunk 1]
```diff
@@ -1,5 +1,5 @@
 NVIDIA NeMo 开发者指南
-====================
+======================
```
Comment (DEKHTIARJonathan) [inline docs/docs_zh/sources/source/index.rst:2]: you should revert this

File: docs/docs_zh/sources/source/index.rst [hunk 2]
```diff
@@ -27,7 +27,7 @@ NeMo 是一个用神经模块来构建 AI 应用的工具包，它与具体的
 
 
 简介
----
+----
```
Comment (DEKHTIARJonathan) [inline docs/docs_zh/sources/source/index.rst:30]: same revert here

File: docs/sources/update_docs_docker.sh [hunk 3]
```diff
@@ -0,0 +1,3 @@
+cd ../../
+docker run --rm -v $PWD:/workspace python:3.7 /bin/bash -c "cd /workspace && pip install -r requirements/requirements_docs.txt && cd docs/sources/ && bash update_docs.sh"
+cd docs/sources
```
Comment (DEKHTIARJonathan) [inline docs/sources/update_docs_docker.sh:3]: you don't need this last line. It's bash script ;)

File: docs/docs_zh/sources/source/conf.py [hunk 4]
```diff
@@ -15,16 +15,14 @@
 
 import os
 import sys
-from unittest.mock import MagicMock
```
Comment (tkornuta-nvidia) [inline docs/docs_zh/sources/source/conf.py:18]: 👍

File: docs/docs_zh/sources/source/conf.py [hunk 5]
```diff
@@ -86,13 +106,15 @@
...
+from package_info import __version__
+
 # The short X.Y version.
 # version = "0.10.0"
-version = str(nemo.__version__)
+version = __version__
 # The full version, including alpha/beta/rc tags.
 # release = "0.9.0"
-release = str(nemo.__version__)
+release = __version__
```
Comment (tkornuta-nvidia) [inline docs/docs_zh/sources/source/conf.py:117]: Fascinating... I thought I have introduced _exactly_ that change in one of my PRs... :)

Comment (tkornuta-nvidia) [reply docs/docs_zh/sources/source/conf.py:117]: ehehehe
https://github.com/NVIDIA/NeMo/blob/64eed9a5980b7f4eeca250e7211d7672fd8fe35c/docs/source/conf.py#L96

yap, I did ;)

Comment (tkornuta-nvidia) [reply docs/docs_zh/sources/source/conf.py:117]: (but in un-merged branch that waits for RTD to FINALLY WORK... :] )

File: docs/sources/update_docs_docker.sh (hunk 3 above)
Comment (blisc) [reply docs/sources/update_docs_docker.sh:3]: thanks, will update in a future pr

**Comment by DEKHTIARJonathan** [comment]
@tkornuta-nvidia it's fine. At least it gives a consistent fix everywhere ;)

**Pull request merged**